ui/app/routes/vault/cluster/clients/counts.ts

```diff
@@ -59,28 +52,19 @@ export default class ClientsCountsRoute extends Route {
     return { activity, activityError };
   }
 
-  async getActivatedFeatures() {
-    try {
-      const resp: ActivationFlagsResponse = await this.store
-        .adapterFor('application')
-        .ajax('/v1/sys/activation-flags', 'GET', { unauthenticated: true, namespace: null });
-      return resp.data?.activated;
-    } catch (error) {
-      if (DEBUG) console.error(error); // eslint-disable-line no-console
-      return [];
-    }
-  }
+  showSecretsSync(activity: ClientsActivityModel | undefined): boolean {
+    // if there is a history of sync clients, show it
+    if (activity && activity?.total?.secret_syncs > 0) return true;
 
-  async isSecretsSyncActivated(activity: ClientsActivityModel | undefined) {
-    // if there are secrets, the feature is activated
-    if (activity && activity.total?.secret_syncs > 0) return true;
+    // otherwise, show the tab based on the cluster type and license
+    if (this.version.isCommunity) return false;
 
-    // if feature is not in license, it's definitely not activated
-    if (!this.version.hasSecretsSync) return false;
+    const isManaged = this.flags.isManaged;
+    const onLicense = this.version.hasSecretsSync;
 
-    // otherwise check explicitly if the feature has been activated
-    const activatedFeatures = await this.getActivatedFeatures();
-    return activatedFeatures.includes('secrets-sync');
```

Comment:
removing this line is what gives HVD clusters access to these pages and charts. now, the new way we allow access is:

```typescript
    // if there is a history of sync clients, show it
    if (activity && activity?.total?.secret_syncs > 0) return true;

    // otherwise, show the tab based on the cluster type and license
    if (this.version.isCommunity) return false;
    if (this.version.isCommunity) return false;

    const isManaged = this.flags.isManaged;
    const onLicense = this.version.hasSecretsSync;

    // we can't tell if HVD clusters have the feature or not, so we show it by default
    // if the cluster is not HVD, show the tab if the feature is on the license
    return isManaged || onLicense;
```

i renamed this to `showSecretsSync` because it doesn't actually check whether the feature is activated. 

TODO:
- show sync charts only if feature is activated

Reply:
I think above some of the conditionals are copy pasta'd but this made sense reading the method! 
And good call on renaming the method!

Reply:
ah, WRT more granular checks i was thinking about this situation:

- no history of sync clients
- not activated

i believe we want to hide the sync client charts in that case because there's nothing to show. this is what's outlined in the tests today, too:
https://github.com/hashicorp/vault/blob/af3e6dd2a71b0b881f81bfcc0be8acdb69ccf292/ui/tests/acceptance/clients/counts/overview-test.js#L281-L291

but yes, your comment on the sync page component is also correct!

Reply:
Okay gotcha. We just want to make sure we don't "hide" the charts based on 0 sync clients (no history of sync clients) so I just wanted to confirm that the empty chart/empty state logic would be staying the same 😄